### PR TITLE
feat: compress declaration

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Compress few popular properties, I will add them more and more in future

**BREAKING CHANGE:**

No

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_css_minifier/src/compress/declaration.rs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                                right.unwrap().clone(),
+                                bottom.unwrap().clone(),
+                            ];
+                        }
+                    }
+                }
+                "font-weight" => {
+                    declaration.value = declaration
+                        .value
+                        .iter()
```
Comment (kdy1) [inline crates/swc_css_minifier/src/compress/declaration.rs:82]: I think we can use `.take().into_iter()` and remove `.clone()` in the closure

**Review by kdy1 — CHANGES_REQUESTED** [review]
Thank you!

File: crates/swc_css_minifier/src/compress/declaration.rs (hunk 1 above)
Comment (alexander-akait) [reply crates/swc_css_minifier/src/compress/declaration.rs]: Done :+1:

**Review by kdy1 — APPROVED** [review]
LGTM. Thank you!

---

swc-bump:
 - swc_css_minifier

**Pull request merged**